airflow/providers/databricks/hooks/databricks.py

```diff
@@ -352,3 +355,44 @@ def get_repo_by_path(self, path: str) -> Optional[str]:
             return str(result['object_id'])
 
         return None
+
+    def import_notebook(self, notebook_name: str, raw_code: str, language: str, overwrite: bool = True, format: str = 'SOURCE'):
```

Comment:
Maybe instead of notebook name we let the user define a notebook path so that they can define the directory structure on the FS.  See comment below as well in the json section.

Reply:
Added and tested. Works great!